app/views/games/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="inner-page-body">
+  <section style="display: flex; flex-direction: row; justify-content: space-evenly; padding-top: 7em">
+    <section>
+      <div><%= image_tag  src="https://cdn11.bigcommerce.com/s-ymgqt/products/24049/images/27742/mario_kart__77494.1432323546.500.750.jpg?c=2", :class => "style_image" %></div>
+      <div> <%= button_to  "Favorite ❤️", "/favorties" %> </div>
```

Comment:
I think this works, because its what we originally talked about. I've since started using a `game controller` icon, and changed the wording to "Saved Games"